mungegithub/submit-queue/www/index.html

```diff
@@ -148,12 +148,21 @@ <h4 class="md-body-2">{{pr.Reason}}</h4>
               <md-toolbar class="md-whiteframe-z2">
                 <h2 class="md-toolbar-tools">End-to-End Results</h2>
               </md-toolbar>
+              <h2 class="md-title" ng-show="cntl.builds.length > 0">Queue-Blocking Builds</h2>
               <md-chips ng-model="cntl.builds" readonly="true">
```

Comment:
Use md-transform-chip to do the filtering here, instead of splitting out builds explicitly. This is also more flexible if there are more categories in the future.

> * md-transform-chip	expression	
> An expression of form myFunction($chip) that when called expects one of the following return values:
> 
> an object representing the $chip input string
> undefined to simply add the $chip input string, or
> null to prevent the 

chip from being appended